tests/test_interfaces.py

```diff
@@ -27,7 +31,8 @@ def test_interfaces(duthost):
         # verify no ipv4 address for each vlan member
         for member in v['members']:
             ans_ifname = "ansible_%s" % member
-            assert not host_facts[ans_ifname].has_key("ipv4")
+            pytest_assert("ipv4" not in host_facts[ans_ifname],
+                          "vlan member {} has IP address {}".format(ans_ifname, host_facts[ans_ifname]))
```

Comment:
for ip addr - host_facts[ans_ifname]["ipv4"]

Reply:
Sure. done.

Reply:
Please see above. There could be key error if we do so.